cypress/e2e/helpers/surveyHelper.ts

```diff
@@ -35,61 +38,129 @@ export const verifySurveyCard = (
   surveyName: string,
   frequency: string,
   schedule: string,
+  status: string,
 ) => {
   cy.contains("h6", surveyName);
   cy.contains("h6", frequency);
   cy.contains("h6", schedule);
+  cy.contains("h6", status);
 };
 
-export const verifyWeightScreenContent = () => {
+export const verifyScreenContent = (
+  progressBarImageSelector: string,
+  percentValue: string,
+  surveyTitle: string,
+) => {
   cy.contains(BUTTON, surveyStrings.CLOSE).should("be.visible");
-  cy.contains("h4", surveyStrings.SERISD001_BODY_WEIGHT).should("be.visible");
+  cy.contains("h4", surveyTitle).should("be.visible");
 
   const currentDate = new Date();
   const formattedDateText = format(currentDate, "EEEE, MMMM dd");
   cy.get(DATE_TEXT).should("contain", formattedDateText);
 
   cy.contains(surveyStrings.WHAT_IS_YOUR_WEIGHT_TODAY);
-  cy.contains(surveyStrings.PLEASE_INSERT_VALUE_BETWEEN_60_700_LBS);
+  cy.contains(surveyStrings.PLEASE_WRITE_FULL_NUMBER_DECIMAL);
   cy.contains(surveyStrings.REQUIRED);
   cy.get(VALUE_LBS_INPUT).should(
     "have.attr",
     "placeholder",
     surveyStrings.LBS_000_0,
   );
-  cy.get(LBS_TEXT).should("contain", surveyStrings.LBS);
+  cy.get(VALUE_TYPE).should("contain", surveyStrings.LBS);
+  cy.contains(surveyStrings.PLEASE_INSERT_VALUE_BETWEEN_60_700_LBS);
 
-  cy.get(PROGRESS_BAR_50_IMG).should("exist");
-  cy.get(VALUE_PROGRESS_BAR_TEXT).should("contain", surveyStrings.PERCENT50);
+  cy.get(progressBarImageSelector).should("exist");
+  cy.get(VALUE_PROGRESS_BAR_TEXT).should("contain", percentValue);
 };
 
-export const verifyWeightIntervalMsgError = (showMsgError: boolean) => {
+export const verifyIntervalMsgError = (
```

Comment:
Now that error message can be any message I think we could give a more generic name
```suggestion
export const verifyInputMessageError = (
```